src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/Assets/js/App.tsx

```diff
@@ -0,0 +1,151 @@
+import React, { useState, useEffect } from "react";
+import ReactDOM from "react-dom";
+import GraphiQL from "graphiql";
+import GraphiQLExplorer from "graphiql-explorer";
+import { buildClientSchema, getIntrospectionQuery } from "graphql";
+
+import "graphiql/graphiql.css";
+import "../css/graphiql-orchard.css";
+
+function fetcher(params: Object): Promise<any> {
+    return fetch(
+        "/api/graphql",
```

Comment:
It needs to be `@Href("~/api/graphql")`, so if it's in a script, you need to generate it in a well-known tag and fetch it from JS.

Reply:
Thanks @sebastienros, I've pushed the fix for this.